scout/supporting_data/config_schema.yml

```diff
@@ -0,0 +1,243 @@
+          tsv_daily_hr_restrict:
+            type: ["string", "null"]
+            enum: [all, peak, low, null]
+            description: The daily hour range to restrict time-sensitive metrics, where `all` is all hours, `peak` is peak demand period hours, and `low` is low demand period hours. Required if running tsv_metrics.
+          tsv_sys_shape_case:
+            type: ["string", "null"]
+            enum: [total reference, total high renewables, net renewable reference, net renewable high renewables, null]
+            description: The basis for determining hours for peak or low demand. Required if tsv_daily_hr_restrict is `peak` or `low`.
+          tsv_season:
+            type: ["string", "null"]
+            enum: [summer, winter, intermediate, null]
+            description: Season of focus for time-sensitve metrics.
+          tsv_energy_agg:
+            type: ["string", "null"]
+            enum: [sum, average, null]
+            description: Define how the the tsv_daily_hr_restrict hours are aggregated, required when tsv_type is `energy`.
+          tsv_power_agg:
+            type: ["string", "null"]
+            enum: [peak, average, null]
+            description: Define how the the tsv_daily_hr_restrict hours are aggregated, required when tsv_type is `power`.
+          tsv_average_days:
+            type: ["string", "null"]
+            enum: [all, weekdays, weekends, null]
+            description: The day type to average over. Only applicable if tsv_type is `energy` or tsv_power_agg is `average`.
+        if:
+          properties:
+            tsv_daily_hr_restrict:
+              enum: [peak, low]
+        then:
+          required: [tsv_sys_shape_case]
+        if:
+          properties:
+            tsv_type:
+              const: energy
+        then:
+          required: [tsv_energy_agg, tsv_average_days]
+        else:
+          if:
+            properties:
+              tsv_type:
+                const: power
+          then:
+            required: [tsv_power_agg]
+            if:
+              properties:
+                tsv_power_agg:
+                  const: average
+            then:
+              required: [tsv_average_days]
+
+      sect_shapes:
+        type: boolean
+        default: false
+        description: Enable sector shapes.
+
+      rp_persist:
+        type: boolean
+        default: false
+        description: Enable relative performance persistence.
+
+      verbose:
+        type: boolean
+        default: false
+        description: Enable verbose mode.
+
+      health_costs:
+        type: boolean
+        default: false
+        description: Enable health costs. Requires alt_regions to be set to `EMM`.
+
+      split_fuel:
+        type: boolean
+        default: false
+        description: Enable split fuel.
```

Comment:
Rephrase this description to something like: "(Enable/disable) ECM results reporting split out by fuel type."